paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -176,52 +198,15 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
         }
 
         return customerRepository.getPaymentMethods(
-            customerConfig,
-            paymentMethodTypes
+            customerConfig = customerConfig,
+            types = paymentMethodTypes,
         ).filter { paymentMethod ->
             paymentMethod.hasExpectedDetails() &&
                 // PayPal isn't supported yet as a saved payment method (backend limitation).
                 paymentMethod.type != PaymentMethod.Type.PayPal
         }
     }
 
-    private suspend fun retrieveSavedPaymentSelection(
-        prefsRepository: PrefsRepository,
-        isGooglePayReady: Boolean,
-        isLinkReady: Boolean,
-        paymentMethods: List<PaymentMethod>
-    ): SavedSelection {
```

Comment:
Here’s a big change: We don’t persist a payment method anymore unless the user has actively selected it. Before, we’d determine a default payment method and persist it immediately.

Reply:
So, if a user never selects a payment method, we will never save it, correct?

Reply:
Yes. We only save if the selection is an intentional action by the user.